client/document/index.jsx

```diff
@@ -156,18 +157,28 @@ class Document extends React.Component {
 					<script
 						type="text/javascript"
 						nonce={ inlineScriptNonce }
+						key={ '__inline_script' }
 						dangerouslySetInnerHTML={ {
 							__html: inlineScript,
 						} }
 					/>
 
-					{ i18nLocaleScript && <script src={ i18nLocaleScript } /> }
-					<script src={ urls.manifest } />
-					<script src={ urls.vendor } />
-					<script src={ urls[ jsFile ] } />
-					{ chunk && <script src={ urls[ chunk ] } /> }
-					<script type="text/javascript">window.AppBoot();</script>
+					{ i18nLocaleScript && <script key={ i18nLocaleScript } src={ i18nLocaleScript } /> }
 					<script
+						key="__manifest"
+						nonce={ inlineScriptNonce }
+						dangerouslySetInnerHTML={ {
+							__html: manifest,
+						} }
+					/>
+					{ entrypoint.map( asset => <script key={ asset } src={ asset } /> ) }
+					{ chunkFiles.map( chunk => <script key={ chunk } src={ chunk } /> ) }
+					<script key="__boot" nonce={ inlineScriptNonce } type="text/javascript">
```

Comment:
Irrelevant [`key`](https://reactjs.org/docs/lists-and-keys.html#keys)? Can we drop it?